src/pages/admin/dashboard/Dashboard.vue

```diff
@@ -1,40 +1,5 @@
-<template>
-  <div class="dashboard">
-    <dashboard-charts />
+<template>TODO</template>
```

Comment:
Todo? Revert?

Reply:
No. We'll scrap Dashboard page. And it had errors, sometimes breaking vue. So I discarded it.